tests/src/JIT/opt/Devirtualization/sealeddefault.cs

```diff
@@ -0,0 +1,28 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+public class Base
+{
+    public virtual int Foo() { return 33; }
+    
+    static BaseSealed s_Default = new BaseSealed();
+
+    public static Base Default => s_Default;
+}
+
+sealed class BaseSealed : Base {}
+
+// The jit ought to be able to devirtualize the call to b.Foo below.
```

Comment:
Does "ought to" mean "in the future should", or "can now"?  If it can now, that's neat, I was expecting the stloc/ldloc through b to block it.  If it can't now but should in the future, then the comment confused me since the comments you've added elsewhere describe the current state.

(where by "current" and "now" I mean with your not-yet-merged devirtualization changes)